extensions/windows-event-log/Bookmark.h

```diff
@@ -21,20 +22,26 @@ class Bookmark
 public:
   Bookmark(const std::wstring& channel, const std::wstring& query, const std::string& bookmarkRootDir, const std::string& uuid, bool processOldEvents, std::shared_ptr<core::CoreComponentStateManager> state_manager, std::shared_ptr<logging::Logger> logger);
   ~Bookmark();
-  operator bool() const;
-  EVT_HANDLE getBookmarkHandleFromXML();
+  explicit operator bool() const noexcept;
+
+  /* non-owning */ EVT_HANDLE getBookmarkHandleFromXML();
```

Comment:
Can it return const to indicate the lack of ownership?

Reply:
How would a const handle indicate lack of ownership? Normally raw pointers should be non-owning and the thought here was that `EVT_HANDLE` is a type alias to `void*`, so I used it indicator of lack of ownership.

This is a bit problematic in our codebase since we can't return owning raw pointers without gsl (that would normally be `gsl::owner<T*>`), but with `unique_ptr` I didn't need it this time.

Reply:
Okay, makes sense. 

Another reason to bring GSL... :)